test/sanity/validate-modules/utils.py

```diff
@@ -57,8 +56,6 @@ def __enter__(self):
         self.sys_stderr = sys.stderr
         sys.stdout = self.stdout = StringIO()
         sys.stderr = self.stderr = StringIO()
-        setattr(sys.stdout, 'encoding', self.sys_stdout.encoding)
-        setattr(sys.stderr, 'encoding', self.sys_stderr.encoding)
```

Comment:
Sorry for this but I thought that this piece of code is used only for module validation and to be honest I still cannot find the connection between `CaptureStd()` and mentioned `get_docstring()`. I also used grep on the whole codebase and I cannot find another usage of this class than the one in validate-modules.

The reason for this change was that `StringIO` from `io` module does not support attributes setting and it is handling everything as unicode which I thought is right.

I'll change it to `io.BytesIO` where encoding attribute setting is possible.